### PR TITLE
generateMode/CopyData

### PR DESCRIPTION
**Pull request by LittleHealth**
Add one generating mode for series: copy data, which copies real data source(.csv) and inserts anomalies to get a very long pseudo-real time series.

### CODE REVIEW

File: core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/source/CopyDataReader.java [hunk 1]
```diff
@@ -0,0 +1,187 @@
+package cn.edu.tsinghua.iotdb.benchmark.source;
```
Comment (stormbroken) [inline core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/source/CopyDataReader.java:1]: you need to add license in new file

**Review by stormbroken — CHANGES_REQUESTED** [review]

**Comment by stormbroken** [comment]
Thanks for your contribution, excellent job!
From my perspective, I have some suggestions.
1. Firstly, I think you can add some doc about how to use these parameters, and you can add them into README.md and README-cn.md for other users who may need these features.
2. Secondly, I want to know  what is the main difference between isIS_COPY_MODE = false and isIS_COPY_MODE = true? From my perspective, now you implementation looks like just do some changes on the data from the csv file, I think you can specific this thing in the doc for other user to choose which one to use. Related things are as follows, how many batch in one file, difference between the timestamp  in the file and and the timestamp finally written into database, difference between the value in the file and the value finally written into database.

File: core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/source/CopyDataReader.java (hunk 1 above)
Comment (LittleHealth) [reply core/src/main/java/cn/edu/tsinghua/iotdb/benchmark/source/CopyDataReader.java:20]: done! Thanks

**Review by stormbroken — APPROVED** [review]
LGTM

**Pull request merged**